pkgs/applications/video/mpv/scripts/autoload.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, mpv-unwrapped, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "mpv-autoload";
+  version = mpv-unwrapped.version;
+  src = "${mpv-unwrapped.src.outPath}/TOOLS/lua/autoload.lua";
```

Comment:
Technically you don't need that anymore. You can use this value directly in `installPhase`.

Reply:
On a second thought, this is more readable.